### PR TITLE
Moving those 2 functions here

### PR DESCRIPTION
**Pull request by MuriloMir**
I've just moved those 2 functions to simpledisplay.

### CODE REVIEW

File: simpledisplay.d [hunk 1]
```diff
@@ -14760,3 +14760,18 @@ class NotYetImplementedException : Exception {
...
+
+version (linux)
+{
+    void moveArrowBy(int x, int y)
+    {
+        XWarpPointer(XDisplayConnection.get(), None, None, 0, 0, 0, 0, x, y);
+        XFlush(XDisplayConnection.get());
+    }
+    
+    void moveArrowTo(int x, int y)
```
Comment (adamdruppe) [inline simpledisplay.d:14772]: this function never uses the x,y params!

IT should just be written in terms of XWarpPointer too.

Comment (MuriloMir) [reply simpledisplay.d:14772]: Ooops, I accidentally forgot to replace the 0, 0 with x, y when I made the experiment to move it to the upper left corner of the screen. That can be easily fixed. But I don't see anything else wrong with this function.

Comment (adamdruppe) [reply simpledisplay.d:14772]: it is just that there's a better way, you can do it in a single call of XWarpPointer by setting the destination window to the root window.

**Comment by adamdruppe** [comment]
ok im gonna hit merge then do some fixes myself, it will be pretty different though

**Comment by adamdruppe** [comment]
here's the way i coded it

https://github.com/adamdruppe/arsd/commit/136b4d0ad50bec88e2601ccb5c9bb53028f8134d

and a usage example:

```

import arsd.simpledisplay;

void main() {
        SyntheticInput i = SyntheticInput(0);

        i.pressKey(Key.Shift, true); // press and hold Shift
        i.pressMouseButton(MouseButton.left, true); // hold left mouse button
        i.moveMouseArrowBy(50, 10);
        i.pressMouseButton(MouseButton.left, false); // release button
        i.pressKey(Key.Shift, false); // release shift

        i.sendSyntheticInput("ok cool"); // send a plain ascii string
}
```

**Comment by adamdruppe** [comment]
by putting it in the object i could load the library without a new build dependency and sort the api out a little. It is also next to the Windows function in the file so if we want to add more Windows functions it can follow the same pattern.

**Pull request merged**